watertap/property_models/multicomp_aq_sol_prop_pack.py

```diff
@@ -382,16 +382,24 @@ def build(self):
             else:
                 self.neutral_set.add(j)
 
+        if len(self.neutral_set)>0 and not len(self.ion_set):
+            _log.warning("Since the charge argument was not provided for any solutes while instantiating the MCAS property model, all solutes are assumed to be neutral and have no charge. If any ions exist in solute_list, make sure to provide charge data to avoid erroneous property calculations.")
         # reference
         # Todo: enter any relevant references
 
+        # Check for molecular weight data
+        if not len(self.config.mw_data):
+            raise ConfigurationError("The mw_data argument was not provided while instantiating the MCAS property model. Provide a dictionary with solute names and associated molecular weights as keys and values, respectively.")
```

Comment:
Note to self: probably should improve this conditional to check that each solute has its own MW value provided. This would only catch the case where _no_ MW data was provided at all but would not catch a partial set of MW data.

Reply:
Addressed this